### PR TITLE
Feature/TR-765/load test taker response in audio PCI

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TR-765

Make sure the `audioRecording` PCI can accept a serialized state or a response as a state.

When reviewing the response, the state is restored from a serialized version, and the object contains an additional level.

To apply the changes, you can either run `taoUpdate` or directly the migration script:
```
php index.php '\oat\tao\scripts\tools\Migrations' -c execute -v 'oat\qtiItemPci\migrations\Version202103181527241465_qtiItemPci'
```

**Note:** due to some bad design in the PCI helpers, we need MathJax to be installed for the unit test to complete

How to test:
- run the unit tests (`cd tao/views/build && npx grunt connect:dev qtiitempcitest`)
- see the main PR on [taoOutcomeUI](https://github.com/oat-sa/extension-tao-outcomeui/pull/374) for a more complete test case

### CODE REVIEW

File: migrations/Version202103181527241465_qtiItemPci.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202103181527241465_qtiItemPci extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Update Liquids interaction';
```
Comment (btamas) [inline migrations/Version202103181527241465_qtiItemPci.php:21]: Wrong interaction name.

**Review by btamas — CHANGES_REQUESTED** [review]
Everything looks good, except migration script description.

File: migrations/Version202103181527241465_qtiItemPci.php (hunk 1 above)
Comment (jsconan) [reply migrations/Version202103181527241465_qtiItemPci.php:21]: oops, good catch!

**Review by btamas — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by anshul119 — APPROVED** [review]
**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**